exchanges/kucoin/kucoin.go

```diff
@@ -311,533 +206,603 @@ func (ku *Kucoin) GetKlines(ctx context.Context, pair, period string, start, end
+	if orderID == "" {
+		return nil, order.ErrOrderIDNotSet
+	}
+	if symbol == "" {
+		return nil, currency.ErrSymbolStringEmpty
+	}
+	if cancelSize == 0 {
+		return nil, fmt.Errorf("%w, cancel size is required", order.ErrAmountBelowMin)
 	}
 	params := url.Values{}
-	params.Set("currency", ccy)
-	var resp []MarginTradeData
-	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, defaultSpotEPL, http.MethodGet, common.EncodeURLValues(kucoinGetMarginTradeData, params), nil, &resp)
+	params.Set("symbol", symbol)
+	params.Set("cancelSize", strconv.FormatFloat(cancelSize, 'f', -1, 64))
+	var resp *CancelOrderByNumberResponse
+	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, cancelSpecifiedNumberHFOrdersByOrderIDEPL, http.MethodDelete, common.EncodeURLValues("/v1/hf/orders/cancel/"+orderID, params), nil, &resp)
 }
 
-// GetIsolatedMarginPairConfig get the current isolated margin trading pair configuration
-func (ku *Kucoin) GetIsolatedMarginPairConfig(ctx context.Context) ([]IsolatedMarginPairConfig, error) {
-	var resp []IsolatedMarginPairConfig
-	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, defaultSpotEPL, http.MethodGet, kucoinGetIsolatedMarginPairConfig, nil, &resp)
+// CancelAllHFOrdersBySymbol cancel all open high-frequency orders (orders created through
+func (ku *Kucoin) CancelAllHFOrdersBySymbol(ctx context.Context, symbol string) (string, error) {
+	if symbol == "" {
+		return "", currency.ErrSymbolStringEmpty
+	}
+	var resp string
+	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, hfCancelAllOrdersBySymbolEPL, http.MethodDelete, "/v1/hf/orders?symbol="+symbol, nil, &resp)
 }
 
-// GetIsolatedMarginAccountInfo get all isolated margin accounts of the current user
-func (ku *Kucoin) GetIsolatedMarginAccountInfo(ctx context.Context, balanceCurrency string) (*IsolatedMarginAccountInfo, error) {
-	params := url.Values{}
-	if balanceCurrency != "" {
-		params.Set("balanceCurrency", balanceCurrency)
-	}
-	var resp *IsolatedMarginAccountInfo
-	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, defaultSpotEPL, http.MethodGet, common.EncodeURLValues(kucoinGetIsolatedMarginAccountInfo, params), nil, &resp)
+// CancelAllHFOrders cancels all HF orders for all symbol.
+func (ku *Kucoin) CancelAllHFOrders(ctx context.Context) (*CancelAllHFOrdersResponse, error) {
+	var resp *CancelAllHFOrdersResponse
+	return resp, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, hfCancelAllOrdersEPL, http.MethodDelete, "/v1/hf/orders/cancelAll", nil, &resp)
 }
 
-// GetSingleIsolatedMarginAccountInfo get single isolated margin accounts of the current user
-func (ku *Kucoin) GetSingleIsolatedMarginAccountInfo(ctx context.Context, symbol string) (*AssetInfo, error) {
+// GetActiveHFOrders obtain all active order lists, and the return value of the active order interface is the paged data of all uncompleted order lists.
```

Comment:
I'm not sure what this is say to make a suggestion sorry. Could you please reword it?